people/ashwin-ramanathan.md

```diff
@@ -0,0 +1,6 @@
+Ashwin Ramanathan
+--------------
+
+![](photos/ashwin.jpg)
```

Comment:
Please rename `ashwin.jpg` to `ashwin-ramanathan.jpg`, there might be other Ashwin's.